tests/test_routes.py

```diff
@@ -195,21 +195,6 @@ def test_active_page_returns_adds_no_meta_with_preview_flag(self):
         soup = BeautifulSoup(response.data, "html.parser")
         self.assertIsNone(soup.find("meta", {"name": "robots"}))
 
-    def test_inactive_page_returns_302(self):
```

Comment:
These tests no longer apply, because discourse-side, people have changed the pages to all active, so to regenerate this cassette we would need to change the status of the engage pages themselves to inactive